src/core_modules/capture-core/components/WidgetEnrollmentEventNew/Validated/validated.epics.js

```diff
@@ -68,10 +68,13 @@ export const saveNewEnrollmentEventEpic = (action$: InputObservable, store: Redu
                 enrollmentId,
                 completed,
             });
+            const orgUnit = serverData.events[0]?.orgUnit || '';
```

Comment:
I see this is an inherited assumption, but why do we ask for the first event in the array? Will there always be one event in this list?

Reply:
Hey, the function [getAddEventEnrollmentServerData](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/DataEntries/SingleEventRegistrationEntry/DataEntryWrapper/DataEntry/epics/getConvertedNewSingleEvent.js#L64) returns the array `events` with one event. Therefore, I think is safe to assume there will be only one event in [serverData.events](https://github.com/dhis2/capture-app/blob/a3f839d02db9066497487e75943f79d4241c1f4f/src/core_modules/capture-core/components/WidgetEnrollmentEventNew/Validated/validated.epics.js#L61)

Reply:
Ok, thanks - just checking :)